src/db/sql-db.ts

```diff
@@ -138,12 +138,16 @@ const init = async (): Promise<DBClient> => {
         const dbUpserts = toDBRecords(tableName, recordUpserts)
         for (const dbUpsert of dbUpserts) {
           // exclude the default timestamp column unless explicitly specified in mergeOptions
-          if (mergeOptions === undefined) {
-            mergeOptions = Object.keys(dbUpsert).filter((value) => { return value !== defaultTimestampColumn });
-          }
+          // if (mergeOptions === undefined) {
+          //   mergeOptions = Object.keys(dbUpsert).filter((value) => { return value !== defaultTimestampColumn });
+          // }
+
+          // remove undefined properties
+          const properValues = _.omitBy(dbUpsert, _.isUndefined)
```

Comment:
The upsert was overriding data in the table, since the ORM was accessing undefined properties of an object. This fixes that by removing all undefined values. If we want to erase a value, we can explicitly set that value to null in the object passed to `upsert`.

I think this also removes the need for the commented out code above ? @puffo

Reply:
Yep, I think that's cool, much cleaner too 👍